### DISCUSSION TITLE
Infinity Sounds v2.0 Released

### DISCUSSION THREAD

**Discussion by skellytz** [Spellhold-Studios/Infinity-Sounds#3] [Announcements]
The new version adds many new features and now also supports BGEE, SoD, BG2EE, and EET.

You can learn more about Infinity Sounds in the [readme](https://skellytz.github.io/docs/readme-infinitysounds.html) and download the mod from [Releases](https://github.com/skellytz/infinity-sounds/releases).

Post your comments about the new version below. If you encounter a problem, report it in [Issues](https://github.com/skellytz/infinity-sounds/issues). Ask a question, share ideas for new features, or make a suggestion in [Discussions](https://github.com/skellytz/infinity-sounds/discussions).

**Comment by mariuszparys** [comment]
Restore Gulp Sound
It seems that this commponent is not working on my installation (Baldur's Gate: Siege of Dragonspear 2.6.6)

// Log of Currently Installed WeiDU Mods
// The top of the file is the 'oldest' mod
// \~TP2_File~ #language_number #component_number // [Subcomponent Name -> ] Component Name [ : Version]
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #110 // Restore BG1 Spellcasting Sounds: v2.0
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #120 // Restore BG1 Weapon Sounds: v2.0
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #130 // Restore BG1 Armor Sounds: v2.0
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #170 // Restore Chunked Death Sound: v2.0
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #180 // Tweak Sound Channel Volume Levels: v2.0
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #189 // Enable Footsteps During Combat: v2.0
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #210 // Restore BG1 Ambient Sounds: v2.0
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #230 // Restore & Tweak Soundsets: v2.0
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #240 // More Existence Sounds: v2.0
\~INFINITYSOUNDS/INFINITYSOUNDS.TP2~ #0 #250 // Tweak Existence Sound Delay: v2.0

**Reply by skellytz** [reply]
Thank you! Described in issue #5.

**Comment by mariuszparys** [comment]
Would you like to share more detailes about which parts of Restore & Tweak Soundsets (_e.g. Completely restored BG1, BG2, and IWD soundsets which previously suffered from animation or filename conflicts, Restored numerous creature sounds, Properly matched attack sound alternatives, Synchronized attack sounds with attack animation frames, Restored many unused NPC lines and sounds, Addressed a lot of minor soundset inconsistencies_) are the most affected by the game engine bugs (_BGEE and BG2EE patch version 2.6_) and to what extent?

Thank you for your work to make this mod compatible with the enhanced edition.

Regards

**Reply by skellytz** [reply]
[Here](https://www.gibberlings3.net/forums/topic/34892-code-code-fixes-aka-stuff-we-cant-fix/?do=findComment&comment=311013) are the issues that were passed on to Beamdog.

EE support is there thanks to other players and modders who have reported bugs or compatibility problems😄 Personally, I play only the classics these days, so I can't possibly notice or keep track of every issue. I'm grateful for all the help!
